src/system/zephyr/network.c

```diff
@@ -404,14 +406,22 @@ int8_t _z_listen_udp_multicast(_z_sys_net_socket_t *sock, const _z_sys_net_endpo
                     if (!mcast) {
                         ret = _Z_ERR_GENERIC;
                     }
+#if KERNEL_VERSION_MAJOR == 3 && KERNEL_VERSION_MINOR > 3
+                    net_if_ipv4_maddr_join(net_if_get_default(), mcast);
```

Comment:
Is there a reason not to reuse `ifa` instead of `net_if_get_default()`?

Reply:
None. Fixed.
And thanks :-)